archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/util/OptionalsValidator.java

```diff
@@ -0,0 +1,348 @@
+/*
```

Comment:
Did you mean to include this file and the test case?

Reply:
This file (along with others) got re-added when I messed-up the merge yesterday.